core/src/main/java/com/nisovin/magicspells/Spell.java

```diff
@@ -902,6 +907,10 @@ protected ConfigData<Component> getConfigDataComponent(String key, Component def
 		return ConfigDataUtil.getComponent(config.getMainConfig(), internalKey + key, def);
 	}
 
+	protected ConfigData<Component> getConfigDataItemComponent(String key, Component def) {
```

Comment:
I don't think this method would be used often enough for it to be more helpful to be here, but it's not a big deal if it is.